src/components/sidepanel/SidePanel.js

```diff
@@ -0,0 +1,32 @@
+import { SidePanelCSS } from "./SidePanelCSS";
+import { colors } from "../colors/Colors";
+import styled from "styled-components"
+
+const SidePanel = (props) => {
```

Comment:
destructure the props here like this
`const SidePanel = ({active, label}) => {`

that way you can use the active/label prop directly without typing props.active each time